.github/workflows/go.yml

```diff
@@ -0,0 +1,50 @@
+name: Go
+
+on:
+  push:
+    branches: [ master, feature/lpos-command ]
+  pull_request:
+    branches: [ master ]
+
```

Comment:
@vmihailenco You don't need to do anything to enable it.

The lines above is where you set the conditions that trigger this action. 

In this case I've set it to whenever there's a `git push` to branches `master` or `feature/lpos-command` . Or when there's a pull request created that targets `master`.

I will remove `feature/lpos-command` from it as it only made sense to my use case. 

I think these are sensible defaults (pull request to master or commits to master) and they are probably the same triggers as travisci.

Reply:
Sounds good - thanks.

Reply:
`feature/lpos-command` is out! 👍